### PR TITLE
PR branch todo1

### PR DESCRIPTION
**Pull request by DarioAlessioR**
PR for activity Give code review, on React tutorial Repo

Project advanced to the middle of Hooks chapter of the tutorial.

### CODE REVIEW

**Comment by maury18** [comment]
# Status approved
Great job Dario, the app works correctly, this PR is approved.
- Great description in the README
- Great optimization of your code 

# Optional Suggestions
It would be better if you could try to add more commits, for the size of this project, it can be helpful in case something goes wrong so you will have better options for a backup

File: src/classBased/components/TodoContainer.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import React from "react"
+import TodosList from "./TodosList"
+import Header from "../../functionBased/components/Header";
+import InputTodo from "./InputTodo";
+import { v4 as uuidv4 } from "uuid";
```
Comment (maury18) [inline src/classBased/components/TodoContainer.js:5]: Great job by importing the react components

**Review by maury18 — APPROVED** [review]
# Status approved
Great job Dario, the app works correctly, this PR is approved.
- Great description in the README
- Great optimization of your code 

# Optional Suggestions
It would be better if you could try to add more commits, for the size of this project, it can be helpful in case something goes wrong so you will have better options for a backup

File: src/classBased/components/TodoContainer.js (hunk 1 above)
Comment (DarioAlessioR) [reply src/classBased/components/TodoContainer.js:5]: Thank you!!!

**Pull request merged**